README.md

```diff
@@ -381,6 +382,49 @@ Stac-server can also be subscribed to SNS Topics that publish complete STAC Item
 
 Errors that occur during ingest will end up in the dead letter processing queue, where they are processed by the `stac-server-<stage>-failed-ingest` Lambda function. Currently all the failed-ingest Lambda does is log the error, see the CloudWatch log `/aws/lambda/stac-server-<stage>-failed-ingest` for errors.
 
+## Pre- and Post-Hooks
+
+Stac-server supports two hooks into the request process: a pre-hook and a post-hook. These are each lambda functions which, if configured, will be invoked by stac-server. It is assumed that the stac-server lambda has been granted permission to invoke these lambda functions, if configured.
+
+### Pre-Hook
+
+If the stac-server is deployed with the `PRE_HOOK` environment variable set to the name of a lambda function, then that function will be called as the pre-hook.
```

Comment:
could you add an example of what the value PRE_HOOK and POST_HOOK are set to look like? It's unclear to me just reading this what the "name" of the lambda is.